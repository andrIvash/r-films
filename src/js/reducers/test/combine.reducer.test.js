import { createStore } from 'redux';
import rootReducer from '../index.js';

describe('changeSearchFilter reducer', () => {
  it('should return the initial state', () => {
    const store = createStore(rootReducer);
    expect(store.getState()).toMatchSnapshot();
  });
});
