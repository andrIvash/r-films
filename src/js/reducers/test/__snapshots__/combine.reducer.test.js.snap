// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changeSearchFilter reducer should return the initial state 1`] = `
Object {
  "changeSearchFilter": Object {
    "searchFilter": "title",
  },
  "changeSearchText": Object {
    "searchText": "",
  },
  "changeView": Object {
    "view": "COMMON",
  },
  "filterData": Array [],
  "items": Array [],
  "itemsHasErrored": false,
  "itemsIsLoading": false,
  "selectDataFilter": Object {
    "selected": "release",
  },
  "selectFilm": Object {
    "posterData": Object {
      "overview": "",
      "poster_path": "",
      "release_date": "",
      "tagline": "",
      "title": "",
      "vote_average": 0,
    },
    "selectedGenre": "Drama",
  },
}
`;
